scripts/ci/run_with_singularity.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+SINGULARITY_BIND="/scratch,/eos,/cvmfs" singularity run /cvmfs/unpacked.cern.ch/gitlab-registry.cern.ch/bendavid/cmswmassdocker/wmassdevrolling\:v5 $@
```

Comment:
should we not use "latest" here instead of v5?

Reply:
So "latest" still points to v4.  v5 is rather pointed to by "testing"

This PR is intentionally being used to test v5 BEFORE we point "latest" to it.

Reply:
Ah okay, so this needs to be updated then whenever a new singularity becomes the new default, or like in this case, before.